qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -541,38 +540,35 @@ def _clean_validate_template(cls, md_template, study_id, restriction_dict,
         # we don't modify the user one
         md_template = md_template.copy(deep=True)
 
+        # In the database, all the column headers are lowercase
+        md_template.columns = [c.lower() for c in md_template.columns]
+        # validating pgsql reserved words not to be column headers
+        current_headers = set(md_template.columns.values)
+        reserved_words = qdb.metadata_template.util.get_pgsql_reserved_words()
+        overlap = reserved_words & current_headers
+        if overlap:
```

Comment:
The problem is that then we will not know which comes from invalid chars or from pgsl without making a lot of changes and I'm not sure is worth it. Do you feel strong about it?